wordcloud/tokenization.py

```diff
@@ -51,17 +53,20 @@ def unigrams_and_bigrams(words, normalize_plurals=True):
     # decount words inside bigrams
     for bigram_string, count in counts_bigrams.items():
         bigram = tuple(bigram_string.split(" "))
-        # collocation detection (30 is arbitrary):
-        word1 = standard_form[bigram[0].lower()]
-        word2 = standard_form[bigram[1].lower()]
+        # either bigram word might be missing if it's a stopword.
+        word1 = standard_form.get(bigram[0].lower())
+        word2 = standard_form.get(bigram[1].lower())
 
-        if score(count, counts[word1], counts[word2], n_words) > 30:
+        # collocation detection (30 is arbitrary):
+        if not word1 or not word2 or score(count, counts[word1], counts[word2], n_words) > 30:
```

Comment:
Sorry, I don't understand this and/or might be missing something obvious. This always evaluated to True if one of them is a stopword, right? But it might still not be a collocation, right?

Reply:
You're correct.  I read up about collocations and have updated the branch.